netket/operator/_local_operator.py

```diff
@@ -37,7 +38,10 @@
 
 
 def is_hermitian(a: np.ndarray, rtol=1e-05, atol=1e-08) -> bool:
-    return np.allclose(a, a.T.conj(), rtol=rtol, atol=atol)
+    if issparse(a):
+        return np.allclose(a.todense(), a.T.conj().todense(), rtol=rtol, atol=atol)
+    else:
+        return np.allclose(a, a.T.conj(), rtol=rtol, atol=atol)
```

Comment:
sparse matrices do not support transpose?

Reply:
just wondering why you need to convert it to dense matrix to check fi transponse conjugate is equal to the matrix itself

Reply:
if you do a + a.T.conj() it's very inefficient because you change the sparsity pattern.

This is probably sub-optimal, but I don't know how to check for hermitianity otherwise...